Day4/src/App.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
can leave  a empty line between library imports ( line 1) and local files import (line 2,3,4) for better readability

Reply:
Done.